moochub-api.v3.yml

```diff
@@ -1458,11 +1458,11 @@ components:
       properties:
         identifier:
           description: 'A license according to https://spdx.org/licenses/ or \"proprietary\".'
-          example: 'https://creativecommons.org/publicdomain/zero/1.0'
+          example: 'CC0-1.0'
           format: iri
```

Comment:
Shouldn't this also go away then?

Reply:
@MaxThomasHPI will add a comment on how the license is meant to be used and will adjust the docs to describe the properties more precisely.

Reply:
Ah, forget it - as just discussed.